modules/nw-metallb-bgppeer-cr.adoc

```diff
@@ -45,6 +45,20 @@ If you use this field, you cannot specify a value in the `spec.peerASN` field.
 |`spec.peerAddress`
 |`string`
 |Specifies the IP address of the peer to contact for establishing the BGP session.
+If you use this field, you cannot specify a value in the `spec.interface` field.
+
+|`spec.interface`
+|`string`
+|Specifies the interface name to use when establishing a session. 
+Use this field to configure unnumbered BGP peering.
+There must be a direct, point-to-point connection between the two BGP peers, and you must enable IPv6 RAs (Router Advertisements).
+If you use this field, you cannot specify a value in the `spec.peerAddress` field.
+The `spec.bgp.routers.neighbors.interface` field is a Technology Preview feature only. 
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. 
```

Comment:
Do we really do that every time we have a TP?

I was thinking having only saying `this is TP (link)`